chainer/functions/pooling/average_pooling_nd.py

```diff
@@ -105,7 +157,7 @@ def create_pool_desc(self):
             cuda.cudnn.cudnn.CUDNN_POOLING_AVERAGE_COUNT_INCLUDE_PADDING)
```

Comment:
(For record) there are two modes available in cuDNN. `CUDNN_POOLING_AVERAGE_COUNT_INCLUDE_PADDING` and `CUDNN_POOLING_AVERAGE_COUNT_EXCLUDE_PADDING`.